ReCaptchaFrontendUi/view/frontend/web/js/reCaptcha.js

```diff
@@ -50,16 +50,19 @@ define(
                     $(window).trigger('recaptchaapiready');
                 }.bind(this);
 
-                element = document.createElement('script');
-                scriptTag = document.getElementsByTagName('script')[0];
+                if (window.addedScriptTag === undefined) {
```

Comment:
Could you rename it to make it no too general?
```suggestion
                if (window.addedRecaptchaScriptTag === undefined) {
```

Reply:
@ihor-sviziev this valiable should be global.

Reply:
Could you explain why?

Reply:
Looks like we can replace global variable on new shared service.

The `reCaptcha` component will have dependency (via "import" by path) on this shared service.
Inside of the new service adding script tag will proceed only one time.

Thanks